### ISSUE TITLE
Input.button is missing some accessibility properties

### ISSUE THREAD

**Issue by mfeineis** [mdgriffith/elm-ui#247]
https://ellie-app.com/9xX76R49YD9a1

**Expected behavior**
Since `Input.button` doesn't use `<button>` nor `<input type=button>` according to [MDN aria button role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) it should be set up to activate on `Space` as well as `Enter`. ~Having the `aria-pressed={true/false}` attribute set is also a good idea for screen readers.~ `aria-pressed` is only for toggle buttons, so it's not applicable here, see [W3C Aria Button](https://www.w3.org/TR/wai-aria-practices-1.1/examples/button/button.html)

> Note: If using role="button" instead of the semantic <button> or <input type="button"> elements, you will need to make the element focusable and have to define event handlers for click and keydown events, including the Enter and Space keys, in order to process the user's input. See the official WAI-ARIA example code.

**Versions**
 - OS: Windows 10
 - Browser chrome
 - Browser Version Version 84.0.4147.89 (Official Build) (64-bit)
 - Elm Version 0.19.1
 - Elm UI Version 1.1.7
